### PR TITLE
Country localization files validation via Travis

### PR DESCRIPTION
**Pull request by zupo**
refs niteoweb/woocart#300
refs niteoweb/woocart#301

### CODE REVIEW

**Comment by dz0ny** [comment]
I don't think python or anything here will be ok unless you provide ways to fix issues. Since most contributors will know PHP, the tools need to be from PHP sphere.

**Comment by zupo** [comment]
We own the repo, we can provide fixes.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+sudo: required
```
Comment (dz0ny) [inline .travis.yml]: sudo: false

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+env:
+  global:
+  - PIPENV_VENV_IN_PROJECT=1
+  - PIPENV_IGNORE_VIRTUALENVS=1
+
+install:
+ - pipenv install --dev --deploy && touch .installed
+
+script:
+ - pipenv run black validate.py --check
```
Comment (dz0ny) [inline .travis.yml]: remove, no python soure here

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ - 3.7
+dist: xenial
+
+env:
+  global:
+  - PIPENV_VENV_IN_PROJECT=1
+  - PIPENV_IGNORE_VIRTUALENVS=1
+
+install:
+ - pipenv install --dev --deploy && touch .installed
```
Comment (dz0ny) [inline .travis.yml]: we don't need pipenv. 
```pip install pre-commit==version --pre```

File: .travis.yml (hunk 1 above)
Comment (zupo) [reply .travis.yml]: sudo:false -> no xenial -> no python 3.7

File: .travis.yml (hunk 2 above)
Comment (zupo) [reply .travis.yml]: I don't understand

File: .travis.yml (hunk 3 above)
Comment (zupo) [reply .travis.yml]: We have way more dependencies than just pre-commit. See `Pipfile.lock`.

File: template.yaml [hunk 4]
```diff
@@ -72,7 +72,7 @@ woo/woocommerce_cod_settings:
     title: 'Cash on delivery' # Translate
     description: 'Pay with cash upon delivery.' # Translate
     instructions: 'Pay with cash upon delivery.' # Translate
-    enable_for_methods: ''
+    enable_for_methods: 'no'
```
[the remark this answers is not in the document]
Comment (dz0ny) [reply template.yaml]: empty string or name of method

[the remark this answers is not in the document]
Comment (dz0ny) [reply template.yaml]: @zupo if you downlaod sql backup for uk-showcase and search for woocommerce_bacs_settings you will se what type is serialized php string.

File: template.yaml [hunk 5]
```diff
@@ -28,7 +28,7 @@ wootax/1: # Standard rate
     shipping: '0'
     order: '0'
     class: 'standard-rate'
-    locations: {  }
+    locations: '{ }'
```
[the remark this answers is not in the document]
Comment (akshitsethi) [reply template.yaml]: @zupo I checked.. `'{  }'` doesn't work.

[the remark this answers is not in the document]
Comment (zupo) [reply template.yaml]: ugh

File: template.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (zupo) [reply template.yaml]: @dz0ny: can we have an enum of acceptable values?

[the remark this answers is not in the document]
Comment (dz0ny) [reply template.yaml]: It can be any plugin name for example, so any string?

File: Countries/United Kingdom/local.yaml [hunk 6]
```diff
@@ -19,7 +19,7 @@ woo/woocommerce_price_num_decimals: '2' # Number of decimals
 # Tax rates
 woo/woocommerce_tax_classes: 'Reduced rate\nZero rate' # If more than one reduced rate add '\nReduced rate 2'
 wootax/1: # Standard rate
-    country: UK
+    country: GB
```
Comment (dz0ny) [inline Countries/United Kingdom/local.yaml]: @gitbot ping @dz0ny tomorrow

Comment (zupo) [reply Countries/United Kingdom/local.yaml]: @dmurko: you used `UK` everywhere, but the list that is linked in `template.yaml` says it needs to be `GB`. Pls check.

Comment (dz0ny) [reply Countries/United Kingdom/local.yaml]: image

Comment (dz0ny) [reply Countries/United Kingdom/local.yaml]: image
image

Comment (dz0ny) [reply Countries/United Kingdom/local.yaml]: image

It's GB :)

**Comment by zupo** [comment]
* replaced `GB` with `UK`
* added HTML linter for legal pages

@dz0ny, @dmurko, @akshitsethi: re-review pls

File: Countries/India/legal-cookies.html [hunk 7]
```diff
@@ -40,14 +40,18 @@ <h2>What are your choices regarding cookies</h2>
...
-	<li>For the Safari web browser, please visit <a href="https://support.apple.</li>com/kb/PH21411?locale=en_US" target="_blank">this page from Apple</a>.</li>
-	<li>For any other web browser, please visit your web browser's official web pages.</li>
+  <li>For the Safari web browser, please visit <a href="https://support.apple.</li>com/kb/PH21411?locale=en_US" target="_blank">this page from Apple</a>.</li>
+  <li>For any other web browser, please visit your web browser's official web pages.</li>
 </ul>
 
 <h2>Where can you find more information about cookies</h2>
 
+<<<<<<< HEAD
 <p>You can learn more about cookies on <a href="http://www.allaboutcookies.org/" target="_blank">AllAboutCookies</a> and <a href="http://www.networkadvertising.org/" target="_blank">Network Advertising Initiative</a>.</p>
```
Comment (akshitsethi) [inline Countries/India/legal-cookies.html]: Git conflict message is added over here.

Comment (zupo) [reply Countries/India/legal-cookies.html]: Tnx!

**Pull request merged**